### ISSUE TITLE
C# 8.0 null-forgiving operator: SA1514 false positive: Element documentation should be preceded with blank line.

### ISSUE THREAD

**Issue by rros** [DotNetAnalyzers/StyleCopAnalyzers#3067]
### Description
When using the null-forgiving operator as per https://docs.microsoft.com/en-us/ef/core/miscellaneous/nullable-reference-types#non-nullable-properties-and-initialization the rule SA1514 is triggered for the documentation header of the next property/method.

### Repro steps
Use the null-forgiving operator like described in the Entity Framework documentation.

```csharp
public class Sample
{
    /// <summary>
    /// Gets or sets the value.
    /// </summary>
    public string SomeString { get; set; } = null!;

    /// <summary>
    /// Gets or sets the value.
    /// </summary>
    public string AnotherString { get; set; } = null!;
}
```

![element_documentation_should_be_preceded_by_blank_line](https://user-images.githubusercontent.com/279664/68662502-d64c8a00-0534-11ea-98fe-e5326073340a.png)

### Expected behavior
The rule SA1514 should not be triggered.

### Actual behavior
The rule SA1514 is triggered.

### Known workarounds
Add an extra newline between the line with with null-forgiving operator.

### Related information
* StyleCop.Analyzers Version: 1.2.0-beta.113
* JetBrains Rider Version: 2019.2.3

**Comment by pantosha** [comment]
I added PR with tests. It seems that issue is not reproduced.
